src/index.html

```diff
@@ -1,17 +1,169 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      id="our-form"
+    >
+      <fieldset class="form-box">
+        <legend>Personal information</legend>
+        <div class="form-filed">
+          <label for="surname">Surname:
+            <input type="text" 
+            name="surname" 
+            id="surname"
+            autocomplete="off" 
+            required
+            >
```

Comment:
Fix formating. Should be:
```suggestion
            <input 
                type="text" 
                name="surname" 
                id="surname"
                autocomplete="off" 
                required
            >
```